bundles/org.openhab.binding.androidtv/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -8,4 +8,27 @@
 	<description>This is the add-on for AndroidTV.</description>
 	<connection>local</connection>
 
+	<discovery-methods>
+		<discovery-method>
+			<service-type>mdns</service-type>
+			<mdns-service-type>_androidtvremote2._tcp.local.</mdns-service-type>
+			<match-properties>
+				<match-property>
+					<name>bt</name>
+					<regex>.*</regex>
+				</match-property>
+			</match-properties>
+		</discovery-method>
+		<discovery-method>
+			<service-type>mdns</service-type>
+			<mdns-service-type>_nv_shield_remote._tcp.local.</mdns-service-type>
+			<match-properties>
+				<match-property>
+					<name>SERVER</name>
+					<regex>^$</regex>
```

Comment:
You expect an empty value for the property SERVER ?
`.*` would be less restrictive.

Reply:
Should you also change the same way for androiddebugbtidge ?